docs/update-review.md

```diff
@@ -9,6 +9,10 @@ We detect two kinds of changes:
 * Package changes: Additions, removals, upgrades, and downgrades
 * Important metadata changes: Changes to the source or dist download URLs or if the commit reference changes for an otherwise unmodified version number
 
+To help you better understand why a package is used in your project, we show in bold package names of direct dependencies,
+dependencies defined in your composer.json file, and in regular font package names of transitive dependencies, dependencies
+of your dependencies.
```

Comment:
```suggestion
For a better overview, packages required directly in your composer.json are highlighted in bold. Other packages in regular font are installed as dependencies of your own composer.json dependencies.
```

Note on that matter, does this correctly highlight packages in bold which replace or provide one of the names in the composer.json?

Reply:
As per internal discussion: not yet but will be added